packages/extension/src/gui.view.ts

```diff
@@ -299,6 +301,10 @@ export class MarqueeGui extends EventEmitter {
       return vscode.commands.executeCommand(command, vscode.Uri.parse(args[0].toString()), options)
     }
 
+    if (args && args.length > 0 && command === 'runme.openAsRunmeNotebook') {
+      return vscode.commands.executeCommand(command, vscode.Uri.file(args[0].replace('file://', '')), options)
```

Comment:
let's be sure this works on Windows

Reply:
tested in on Windows, found some minor details but overall it worked.